inbm-lib/tests/unit/inbm_lib/test_trtl.py

```diff
@@ -158,7 +158,7 @@ def test_trtl_image_remove_by_id(self):
             runner, "docker").image_remove_by_id(
             "abc123", True)
         self.__check_trtl_output("/usr/bin/trtl -type=docker "
-                                 "-cmd=imageRemoveByID -id=abc123 -f=True",
+                                 "-cmd=ImageRemoveAll -in=abc123 -f=True",
```

Comment:
Same comment here regarding -id vs -in

Reply:
image_id is not an id, it is a Name of Image. command ImageRemoveAll takes the image name as a parameter, based on that we changed it.